### DISCUSSION TITLE
Add world alias for chat format like EssentialsChat plugin

### DISCUSSION THREAD

**Discussion by sakurawald** [Zrips/CMI#7243] [Feature Requests]
The EssentialsChat plugin provides a build-in world-alias for chat-format.
so that you can set chat-format with %world_alias%, not only the %world_real_name%.
this is useful to hide the long and boring world real name.

**Comment by mrfloris** [comment]
Just to help you resolve this, I use Multiverse-Core for my servers, and many do, maybe you as well. I use the `%multiverse_world_alias%` placeholder from it to show the worlds the way that I want. So I can make Halloween prefixes show in orange during the event for example.

**Reply by sakurawald** [reply]
Haha, i just remove Multiverse-Core some days ago. since most functions of this plugin i never use.
but thank you.

**Reply by mrfloris** [reply]
Which world manager do you use now if you use more than the 1 default world + nether/end ?

**Comment by mrfloris** [comment]
By the way, i just happen to scroll through placeholders, I now see `%cmi_user_world_formatted%` - erhm, would that do the trick perhaps? (haven't tested)

**Reply by sakurawald** [reply]
> cmi_user_world_formatted

not, this placeholder means where the user is on.
but the key problem is that: i want to set some ALIAS FOR WORLDS, so you can hide the real and long world names.
for example, you can show world name "world", "world_nether", "world_the_end" to alias like "survival-world" or something else.
i just very strange taht CMI doesn't have this. since EssentialsChat provides 2 placeholders: %world_name% and %world_name_alias%

**Reply by mrfloris** [reply]
Yeah i was hoping there's a difference between:

```
%cmi_player_world%
and
%cmi_user_world_formatted%
```

If one is the plaintext, then what makes the other "formatted"? The information says this: "Current players world name by using custom identification", sounds promising; what is custom identification.

**Reply by sakurawald** [reply]
the problem is that: i can't found where to set an alias for a world by CMI.
so i can't understand what %cmi_user_world_formatted% means.

**Reply by mrfloris** [reply]
I can't find it either, so, we tried .. back to beginning we go then; requesting a way to set an alias for world, if there's no world manager instealled.

**Reply by sakurawald** [reply]
i did not install any world-manager, since i only use very little functions of them. about 95%-99% functions of them i will not use.

**Reply by sakurawald** [reply]
I have solve this in an in-direct way by CMI.
that's a combo with CMI and LuckPerm.
you can set a world-permission context in LuckPerm, and use ChatGroupFormat by CMI

**Comment by ranyouHigh** [comment]
World aliases can also be modified in the CMILib config file.,which is applied to **%cmi_user_world_formatted%**.
